integrated_channels/cornerstone/views.py

```diff
@@ -127,8 +127,14 @@ def get(self, request, *args, **kwargs):
 
         get_count = int(request.GET.get('count', exporter.MAX_PAYLOAD_COUNT))
         content_item_count = min(get_count, exporter.MAX_PAYLOAD_COUNT)
-        created, updated, deleted = transmitter.transmit(
-            *exporter.export_for_web_polling(max_payload_count=content_item_count)
+        # Remove content records `marked for transmission` flags
+        created, updated, deleted = exporter.export_for_web_polling(max_payload_count=content_item_count)
+
+        create_payload = {record.content_id: record for record in created}
```

Comment:
ah. this should be inside the `export_for_web_polling()` method. the export signature should be consistent - including the structure of its return values.

Reply:
done